### ISSUE TITLE
favicon

### ISSUE THREAD

**Issue by charles-hood** [TheRobBrennan/woatw#17]
Great site, I've been following your development. I also use gatsby-theme-musician and I really like the customization you've done to it. One thing I've noticed however is that you guys are still using the default favicon. When people have lots of browser tabs open, having a custom favicon really helps yours stand out. It also helps with consistency of overall branding, etc.  
It's really easy to customize this to your own logo using gatsby-plugin-favicon. Check it out.

**Comment by TheRobBrennan** [comment]
Hiya @charles-hood! Thanks for following along 😁

You know, it's funny - I wasn't even thinking about that favicon (probably because it's such an easy thing to gloss over incorporating a previously existing theme).

Great catch, man. I'll definitely tweak that in an upcoming PR.

Thanks for keeping me on my toes!

**Comment by TheRobBrennan** [comment]
Favicon has been added as of the above PR. I looked at `gatsby-plugin-favicon` - however it failed to install in Docker. I saw that there was an open PR for that plugin at https://github.com/Creatiwity/gatsby-plugin-favicon/pull/65 - however, it was from October 2019 and still not accepted/merged into `master`

That wound up being a huge red flag for me.

However...Given that I wanted to leverage some of the PWA features down the line, this wound up being a great time to use a default config of `gatsby-plugin-manifest` - with the default config suggested by https://www.gatsbyjs.org/docs/add-a-manifest-file/#using-gatsby-plugin-manifest

Thanks again for your eagle eye on this one. Issue resolved 😁

**Comment by charles-hood** [comment]
Wow, that looks much more simple than the approach I took. I might revert my changes in Github and do it your way. I noticed that gatsby-plugin-favicon pulls in a ton of other stuff, it seems very "heavy." And I know what you mean about red flags. Especially after our experience with the author of gatsby-theme-musician being missing in action, and the show date issue.

**Comment by TheRobBrennan** [comment]
Yeah...I dig the theme, though, and really would like to fork it and make some changes to incorporate the fixes...and a few changes to how things are organized, personal preferences, etc.

I imagine I'll incorporate that in the #100DaysOfGatsby I'm participating in...no ETA, just soon-ish haha

**Comment by charles-hood** [comment]
Yes, I *love* gatsby-theme-musician because it provides the basics of what every musician or band needs, in a simple but good looking format. Customizable, without being overwhelming to newcomers. I would love to see it *actively* maintained though. You should totally fork it. There are some weird design choices that were made. Like the hardcoded file names for artist-banner, etc.

**Comment by charles-hood** [comment]
I run RockOfPages.com in my spare time, as a hobby, where I help local artists put together their web presence. I do this as a way to "give back" to the local music community that brings me so much joy. I currently have four artists online, all using gatsby-theme-musician and Netlify, for no monthly costs. Like you, I've discovered some of the shortcomings in the theme, like how the navigation menu looks on mobile if you add too many items. Unfortunately, I don't have the tech skills to fix it. I'm more of an integrator than a programmer. Check out https://www.thebluebirdtwo.com/ and https://thomashindsmedia.com/ to see some of my sites. The second one uses the MailChimp integration that I learned how to do from watching your site. (Thank you!)
